CHANGELOG.md

```diff
@@ -2,11 +2,18 @@
 
 ## master / unreleased
 
-## 1.4.0-rc.0 / 2020-09-15
+## 1.4.0-rc.1 / 2020-09-21
 
 * [CHANGE] TLS configuration for gRPC, HTTP and etcd clients is now marked as experimental. These features are not yet fully baked, and we expect possible small breaking changes in Cortex 1.5. #3198
+* [CHANGE] Fixed store-gateway CLI flags inconsistencies. #3201
+  - `-store-gateway.replication-factor` flag renamed to `-store-gateway.sharding-ring.replication-factor`
+  - `-store-gateway.tokens-file-path` flag renamed to `store-gateway.sharding-ring.tokens-file-path`
+* [BUGFIX] Handle hash-collisions in the query path. #3192
```

Comment:
Try to express what the user will see.  E.g. "Cortex could occasionally mix up two different series in a query, leading to invalid results, when streaming was used"

Reply:
Good point, done, thanks!